pegasus/sites.v3/code.org/public/css/common-international.css

```diff
@@ -252,35 +244,43 @@
   }
 }
 
-@media screen and (min-width: 600px) /* and (max-width: 969px) */ {
```

Comment:
i think we should probably start adding detailed comments to our css files at some point.. carousels especially are complex!  (or maybe we should just switch to react :)